vendor.conf

```diff
@@ -25,7 +25,7 @@ github.com/RackSec/srslog 456df3a81436d29ba874f3590eeeee25d666f8a5
 github.com/imdario/mergo 0.2.1
 
 #get libnetwork packages
-github.com/docker/libnetwork b13e0604016a4944025aaff521d9c125850b0d04
+github.com/docker/libnetwork 5dc95a3f9ce4b70bf08492ca37ec55c5b6d84975
```

Comment:
for those who need a compare:

Proposed commit
https://github.com/docker/libnetwork/compare/b13e0604016a4944025aaff521d9c125850b0d04...5dc95a3f9ce4b70bf08492ca37ec55c5b6d84975

Master:
https://github.com/docker/libnetwork/compare/b13e0604016a4944025aaff521d9c125850b0d04...master

Is there something I'm missing?